ref/general/securing-microservices.adoc

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2019 IBM Corporation and others.
+// Licensed under Creative Commons Attribution-NoDerivatives
+// 4.0 International (CC BY-ND 4.0)
+//   https://creativecommons.org/licenses/by-nd/4.0/
+//
+// Contributors:
+//     IBM Corporation
+//
+:page-description: Securing cloud-native applications is an important part of your development process as applications move from monolithic to microservices architecture.
+:seo-title: Securing cloud-native microservices
+:seo-description: Securing cloud-native applications is an important part of your development process as applications move from monolithic to microservices architecture.
+:page-layout: general-reference
+:page-type: general
+= Securing cloud-native microservices
+
+As applications move from monolithic to microservices architecture, securing cloud-native applications is an important part of your development process.
+Rather than being an afterthought, security needs to take precedence in the development cycle.
+There are several ways to ensure the security of your cloud-native microservices.
+
+== Defense in depth
+
+A simple firewall does not provide adequate security in a microservices context.
+You need to apply multiple layers of security to ensure that your application is safe.
+In a complex and distributed microservices environment, where multiple processes exist, a single point of security at the network level is inadequate.
+Processes like authentication and authorization, hashing passwords, and encryption provide defense in depth security.
+
+
+== The changed scenario
```

Comment:
this heading could tell you more of what the section is about. As is, it seems indistinct